Packs/CommonScripts/Scripts/ShowLocationOnMap/README.md

```diff
@@ -0,0 +1,23 @@
+Displays a geographical location in an indicator field using Google Maps.
+
+You need to [Setup Google Maps](https://xsoar.pan.dev/docs/reference/articles) to work with Cortex XSOAR. You can then add this automation to an indicator layout. When viewing the indicator you can see the location in the indicator field.
```

Comment:
I'd suggest moving the details about adding a General Purpose Dynamic Section to the indicator layout from https://github.com/demisto/content-docs/pull/919 to this page, because these steps are specific to the `ShowLocationOnMap` automation.

Reply:
Initially I was going to place the content in the `ShowLocationOnMap` automation but there seems to be a convention not to include it. None of the automation docs have additional information. I'm reluctant to add it, so that is why I thought of an additional doc and we'll have an xref to this one.